near/res/testnet_genesis_hash

```diff
@@ -1 +0,0 @@
-ENiDAjkhBgRiC2TNkF9erqiC1DhNVKBug1vNhHSdnfdh
```

Comment:
how come we don't store genesis hash anymore?

Reply:
genesis file download and version check happens at nearup side, which will ensure it download the latest genesis when it detect version has changed (by download a small file contains only genesis_time). So nearcore is run with correct genesis